### ISSUE TITLE
Navigating using Right/Left arrow keys in calendar control, Keyboard focus is not visible on Dates in High Contrast Black/White mode

### ISSUE THREAD

**Issue by ylux587** [telerik/kendo-ui-core#5901]
### Reproduction of the problem
Navigating using Right/Left arrow keys in calendar control, Keyboard focus is not visible on Dates in High Contrast Black/White mode

### Current behavior
Navigating using Right/Left arrow keys in calendar control, Keyboard focus is not visible on Dates in High Contrast Black/White mode

### Expected/desired behavior
Keyboard focus should be visible on Dates in High Contrast Black/White mode

### Environment

* **Kendo UI version:** Kendo UI version: 2019.3.1114
* **jQuery version:** 3.3.28
* **Browser:**Chromium Edge Version: 84.0.488.1 (Official build) dev (64-bit)
* **AI4D Version:** 1.1.1152.1

**Comment by dmanova** [comment]
Hello @ylux587. 
I check the Kendo UI version 2019.3.1114 and it seems it is an Internal Build version, but not an Official Release. 
I would suggest installing the latest Stable (Official) release that is 2020.2.617. Or you could try the ones that are close to your current one (2019.3.1023 or 2020.1.114).
Indeed on some websites the Kendo UI Core Internal Builds have not been marked appropriately with the "-internal" suffix. I would suggest checking this link for official releases - https://www.telerik.com/support/whats-new/kendo-ui/release-history#jquery-release-history

Here is a screenshot of how the calendar looks at my side with Kendo UI version 2019.3.1023, High Contrast Theme, Grayscale screen mode:
![Calendar-Grayscale-HighContrastTheme](https://user-images.githubusercontent.com/52446546/87922971-66a87a80-ca85-11ea-8e5d-fb6bd94af43f.PNG)

The selected date seems eligible. Only the current date seems not well visible when not selected. 

I'm sorry for the inconvenience this might have caused you. 
Please, attach a screenshot if the issue persist with the latest official release.

**Comment by dmanova** [comment]
Rising priority due to ticket 1505283.

**Comment by Dimitar-Goshev** [comment]
R1 2023 is the last official release of Kendo jQuery, where Less Themes are supported and shipped with the product.
